hack/verify-gofmt.sh

```diff
@@ -52,3 +52,14 @@ if [[ -n "${bad_files}" ]]; then
   echo "${bad_files}"
   exit 1
 fi
+
+# Find comments not prefaced with a space.  gofmt doesn't do this.
+# Multiple grep statements is easier to extend debug, although not elegant.
+bad_files=$(find_files | xargs grep '//[^ ]' | grep -v "://" | grep -v swagger | grep -v "\"")
```

Comment:
i think it was ~300+ violations in the code base.   i can double check and make sure they are all "true" violations.